src/components/organisms/Chat.stories.js

```diff
@@ -20,7 +21,7 @@ class ExampleAgent extends Component {
     EventHandler.subscribe(EVENT_USER_MESSAGE, message => this.handleHumanChatMessage(message));
 
     chat.addMessages({
-      Component: props => <div>{props}</div>,
+      Component: ChatBubble,
```

Comment:
This prop name is conflicting with the import of Component from react, change name perhaps?

Reply:
A solution could also be to use React.Component instead of importing the Component module